src/main/cpp/Robot.cpp

```diff
@@ -7,19 +7,20 @@
 #include <frc/smartdashboard/SmartDashboard.h>
 #include <frc2/command/CommandScheduler.h>
 
-
-Robot::Robot() {
+Robot::Robot()
+{
+  //we create a new pointer of an object of the RobotContainer class, almost all of this code should already be here
+  //Except for the autnmous stuff
```

Comment:
Instead of putting comments like this into the function itself, let's start shifting towards comments above the function.

[Google's C++ guide](https://google.github.io/styleguide/cppguide.html#Function_Comments) has a decent description of a minimal function comment.  The [JSF style guide](https://stroustrup.com/JSF-AV-rules.pdf) also has a section on comments, and specifically rules 129-131 are good to note.